README.md

```diff
@@ -13,7 +13,7 @@ import FluentSQLiteDriver
 import HummingbirdFluent
 
 let logger = Logger(label: "MyApp")
-let fluent = HBFluent(logger: logger)
+let fluent = Fluent(logger: logger)
```

Comment:
Fluent seems a very generic name here

Reply:
I agree, though I'm not sure of a better name